### PR TITLE
NCSU ECP and Basis set 

### PR DESCRIPTION
**Pull request by anbenali**
Updating the ECPs and basis sets from NCSU. 

In the case of Na Mg Al Si P S Cl Ar,  An He core was available and was selected.

### CODE REVIEW

File: data/pseudo/ncsu [hunk 1]
```diff
@@ -1,56 +1,46 @@
...
 3
--10.851924053 2 21.7769665504
-1.0 1 21.2435950826
-21.2435950826 3 21.2435950826
+1.00000000000000    1 21.24359508259891 
+21.24359508259891   3 21.24359508259891 
+-10.85192405303825  2 21.77696655044365 
 1
-0.0 2 1.0
+0.00000000000000    2 1.000000000000000
```
Comment (TApplencourt) [inline data/pseudo/ncsu]: Why the H changed?

**Comment by TApplencourt** [comment]
Thanks! 
Just a quick question before accepting the change, why some value changed from the one we got previously in QP?  For example H and CU  in the ncsu file

**Comment by anbenali** [comment]
Because the original version was still on arxiv. Now they published all ECP
and basis sets so I replaced with published data instead.

Also, original ecps worked with BFD basis sets. The new basis sets match
the ecp..

**Comment by TApplencourt** [comment]
Thanks!

**Pull request merged**